packages/esm-home-app/src/dashboard/home-dashboard.component.tsx

```diff
@@ -23,7 +23,7 @@ export default function HomeDashboard() {
         {isDesktop(layout) && <ExtensionSlot name="home-sidebar-slot" key={layout} />}
         <DashboardView title={activeDashboard?.name} dashboardSlot={activeDashboard?.slot} />
       </section>
-      <WorkspaceContainer overlay contextKey="home" />
+      <WorkspaceContainer contextKey="home" />
```

Comment:
This change will be made for all the workspaces shown on the `/home` path, which also includes Wards Workspaces and others.

Reply:
@vasharma05, You are right, and I raised the same question during the coffee break (10 Dec 2024) while we were debugging the issue. My suggestion then was to avoid affecting other workspaces by ensuring that each workspace has its own WorkspaceContainer component, with an overlay on top of the content to simplify styling without affecting other workspaces. The main essence of pushing this PR was to get members' views after implementing what @brandones had suggested on the ticket.